ThePhasesOfCraftship/2_best_practice_first/testingBasics/exercises/3_Manage_Complexity/3_2_Boolean_Calculator/src/Evaluateable.ts

```diff
@@ -0,0 +1,5 @@
+export abstract class Evaluateable {
+    abstract evaluate(): boolean;
+
+    abstract toString(): string;
```

Comment:
I only created this to make testing easier.
Using jest's `toMatchObject` function gave some false-positives ex:
`expect(new And(TRUE, TRUE)).toMatchObject(new Or(TRUE,TRUE)))` would pass, since the jsonified objects jest uses to for comparison would be the same.